modules/downloading-manifest-files-resources-oci.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="downloading-manifest-files-resources-oci_{context}"]
+= Downloading manifest files and deployment resources
+
+You must download the archive file that includes files for creating cluster resources and custom manifests. After you extract the contents of the archive file, you must upload the extracted files to an {oci-first} stack. A stack is an {oci} feature where you can automate the provisioning all the necessary {oci} infrastructure resources, such as the custom image, that are required for installing a {product-title} cluster on {oci}.
+
+.Prerequisites
+
+* You uploaded a generated Agent ISO image to {oci}.
+* You have permissions to access the `oracle-quickstart / oci-openshift` GitHub repository.
+
+.Procedure
+
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, click the **<> Code** button and then click **Download ZIP**.  The following list details these resources:
++
+** link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests[CCM and CSI custom manifests].
+** link:https://github.com/oracle-quickstart/oci-openshift/blob/main/README.md[Download the {oci} Resource Manager configuration to provision resources for deploying Openshift on {oci}].
++
+[NOTE]
+====
```

Comment:
@faermanj . Do the example configurations refer to https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests/manifests ?